packages/rum/src/domain/record/elementsScrollPositions.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { createElementsScrollPositions } from './elementsScrollPositions'
+
+describe('elementsScrollPositions', () => {
+  beforeEach(() => {
+    document.body.style.setProperty('height', '5000px')
+    document.body.style.setProperty('width', '5000px')
+    window.scroll(10, 20)
+  })
+
+  afterEach(() => {
+    window.scroll(0, 0)
+    document.body.style.removeProperty('height')
+    document.body.style.removeProperty('width')
+  })
+
+  it('should attach document scroll positions to document scrolling element', () => {
+    const elementsScrollPositions = createElementsScrollPositions()
+
+    elementsScrollPositions.set(document, { scrollLeft: 10, scrollTop: 20 })
+
+    expect(elementsScrollPositions.get(document.scrollingElement!)).toEqual({ scrollLeft: 10, scrollTop: 20 })
+  })
+
+  it('should attach document scroll positions to unavailable document scrolling element', () => {
+    const documentScrollingElement = document.scrollingElement!
+    Object.defineProperty(document, 'scrollingElement', { value: null, configurable: true })
+    const elementsScrollPositions = createElementsScrollPositions()
+    elementsScrollPositions.set(document, { scrollLeft: 10, scrollTop: 20 })
+
+    expect(elementsScrollPositions.get(documentScrollingElement)).toEqual({ scrollLeft: 10, scrollTop: 20 })
+    Object.defineProperty(document, 'scrollingElement', { value: documentScrollingElement, configurable: true })
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Restore original scrollingElement in an `afterEach` callback to make sure it is correctly restored